DOC-11938 RBAC-API-for-creating-external-user-is-wrong-in-the-lookup-page

supritha-kumar opened the pull request:
'/settings/rbac/users/external/<new-username>' needs to be updated at https://docs.couchbase.com/server/current/rest-api/rbac.html#create-an-external-user-and-assign-roles

supritha-kumar committed:
Update rbac.adoc

supritha-kumar committed:
Updated rest-security-authorization-table.adoc

changed "/settings/rbac/users/local/<new-username>" to "/settings/rbac/users/external/<new-username>"

ggray-cb commented on modules/rest-api/pages/rbac.adoc:
```diff
@@ -431,7 +431,7 @@ If successful, the call creates local user `sdavis` and adds them to the `Cluste
 === Create an External User, and Assign Roles
 
-To create an external user, and assign them one or more roles, use the `PUT` method with the `/settings/rbac/users/external/<new-username>` URI.
+To create an external user, and assign them one or more roles, use the `PUT` method with the `/settings/rbac/users/local/<new-username>` URI.
```
I believe this is wrong. This is discussing creating an external user. The /external/ should be part of this path. I believe all you need is the other change in the PR.

supritha-kumar replied:
Have reverted the change.

supritha-kumar committed:
Updated rbac.adoc

reverted change as per https://github.com/couchbase/docs-server/pull/3882/files
